plugins/module_utils/spec.py

```diff
@@ -41,11 +42,18 @@ def __init__(
         search_path=None,
         doctype="config",
         keep_tmp_files=False,
+        xml_schema=None,
```

Comment:
Passing `xml_schema`, `json_schema` and `tree_schema` is not required

Reply:
Updated